src/lib.rs

```diff
@@ -113,13 +113,15 @@ pub use miniscript::Miniscript;
 pub trait MiniscriptKey:
     Clone + Eq + Ord + str::FromStr + fmt::Debug + fmt::Display + hash::Hash
 {
+    const IS_BITCOIN_PUBKEY: bool = false;
```

Comment:
I think it would be better to have method `is_uncompressed()` which has a default implementation which returns false. Then **only** for `bitcoin::PublicKey` do we implement this and have it actually return the uncompressedness.